CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 <!-- <START NEW CHANGELOG ENTRY> -->
 
+## 5.8.0
```

Comment:
The changelog is updated automatically during the release process, this change would interfere with that logic.

Reply:
Ah cool! Thought it might be the case but wasn't sure.